ccloud/docs/beginner-cloud.rst

```diff
@@ -1,46 +1,43 @@
-
-.. _beginner-cloud:
+.. _ccloud-cli-tutorial:
 
 Tutorial: |ccloud| CLI
 =======================
 
 Overview
 --------
 
-This tutorial will show you how to use the `Confluent Cloud CLI
+This tutorial shows you how to use the `Confluent Cloud CLI
 <https://docs.confluent.io/current/cloud/cli/install.html>`__ to interact with
 your `Confluent Cloud <https://confluent.cloud/login>`__ cluster.
+Following the workflow in this tutorial, you complete the following steps:
 
-With :devx-examples:`start.sh|ccloud/beginner-cloud/start.sh`, a fully scripted
-demo that shows you how to interact with |ccloud|, you can complete the
-following workflow in about 8 minutes using the |ccloud| CLI:
-
--  `Create a new Confluent Cloud environment and specify it as the default`_
--  `Create a new Confluent Cloud cluster and specify it as the default`_
--  `Create a new API key/secret pair for user and specify it as the default`_
+-  `Create a new Confluent Cloud environment`_
+-  `Create a new Confluent Cloud cluster`_
+-  `Create a new API key/secret pair for user`_
 -  `Produce and consume records with Confluent Cloud CLI`_
 -  `Create a new service account with an API key/secret pair`_
--  `Run a Java producer before and after configuring the ACLs`_
--  `Run a Java producer to show a prefixed ACL`_
--  `Run Connect and kafka-connect-datagen connector with permissions`_
--  `Run a Java consumer to show a Wildcard ACL`_
--  `Delete the API key, service account, Kafka topics, Kafka cluster, environment,
-   and the log files`_
+-  `Run a Java producer without ACLs`_
+-  `Run a Java producer with ACLs`_
+-  `Run a Java producer with a prefixed ACL`_
+-  `Run kafka-connect-datagen connector with wildcard ACLs`_
+-  `Run a Java consumer with a Wildcard ACL`_
+-  `Clean up your Confluent Cloud resources`_
 
 
 Prerequisites
 ~~~~~~~~~~~~~~
 
--  Access to a |ccloud| cluster. See `Create a Cluster in Confluent Cloud
-   <https://docs.confluent.io/current/cloud/clusters/create-cluster.html>`__
-
--  Local `install of Confluent Cloud CLI
-   <https://docs.confluent.io/current/cloud/cli/install.html>`__ (v1.7.0 or later)
+-  Access to `Confluent Cloud <https://confluent.cloud/login>`__. Note that this
+   tutorial uses real resources in |ccloud|, and it creates and deletes
+   topics, service accounts, API keys, and ACLs.
```

Comment:
```suggestion
-  Access to `Confluent Cloud <https://confluent.cloud/login>`__.
```

Reply:
Moved the note caveat to the intro section^^

Reply:
Ack and done